### PR TITLE
made quick amounts dymanic

### PR DESCRIPTION
**Pull request by VikChawla**
**Description**

_Write a short description of the changes included in this pull request._

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

**Issue**

Progresses #???

### CODE REVIEW

File: app/components/UI/FiatOnRampAggregator/Views/AmountToBuy.js [hunk 1]
```diff
@@ -634,17 +645,10 @@ const AmountToBuy = () => {
...
-                  { value: 300, label: '$300' },
-                  { value: 400, label: '$400' },
-                  // eslint-disable-next-line no-mixed-spaces-and-tabs
-                ]
-              : []
-          }
+          amounts={limits?.quickAmounts.map((limit) => ({
+            value: limit,
+            label: currentFiatCurrency?.denomSymbol + limit.toString(),
+          }))}
```
Comment (wachunei) [inline app/components/UI/FiatOnRampAggregator/Views/AmountToBuy.js:651]: Add a default empty array so `QuickAmounts` does not crash.
```suggestion
          amounts={limits?.quickAmounts.map((limit) => ({
            value: limit,
            label: currentFiatCurrency?.denomSymbol + limit.toString(),
          })) ?? []}
```

Comment (VikChawla) [reply app/components/UI/FiatOnRampAggregator/Views/AmountToBuy.js:651]: All other options except for US, Pounds, and Euros will return an empty quickAmounts array. Should I still make that fallback?

Comment (VikChawla) [reply app/components/UI/FiatOnRampAggregator/Views/AmountToBuy.js:651]: > Add a default empty array so `QuickAmounts` does not crash.

All other options except for US, Pounds, and Euros will return an empty quickAmounts array. Should I still make that fallback?

Comment (wachunei) [reply app/components/UI/FiatOnRampAggregator/Views/AmountToBuy.js:651]: Oh nice, then we don't 👌

**Review by wachunei — APPROVED** [review]

**Pull request merged**